### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by ChuaZenKhoon**
Also discovered bug for command message
Fixed minor UG problem

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -454,32 +455,74 @@ testers are expected to do more *exploratory* testing.
...
 
-1. _{ more test cases …​ }_
 
 ### Adding a contact
 
+For manual testing of other commands, run the following commands to add the specified few contacts
+(this is assuming the default tag list given is not edited in any way):
+  1) `+ /name John Doe /id johndoe61 /hp 98765432 /tag finance`
+  2) `+ /name Ali /id ali21 /hp 99795422 /tag sales`
+  3) `+ /name Rose /id rose84 /hp 98631422 /tag RnD`
```
Comment (ShanylOng) [inline docs/DeveloperGuide.md:459]: Thanks for adding the detailed prerequisite steps. LGTM.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -454,32 +455,74 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Prerequisites:
+    * Clear all contacts using the `clear` command.
+    * Ensure the tag `RnD` exists in the tag list by running `ls -t`
+
+    1. Test case: `+ /name Rose S/O Dan /id rose84 /hp 98631422 /tag RnD` <br>
+       Expected: No contact is added. Error details shown in the message box.
+    2. Test case: `+ /name Rose-Dan /id rose84 /hp 98631422 /tag RnD` <br>
+       Expected: No contact is added. Error details shown in the message box.
+
```
Comment (ShanylOng) [inline docs/DeveloperGuide.md:483]: LGTM.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -454,32 +455,74 @@ testers are expected to do more *exploratory* testing.
...
+    1. Prerequisites:
+    * Clear all contacts using the `clear` command.
+    * Add a few contacts in according to [this section](#adding-a-contact).
+    * The following is done with the default tag list.
+
+    1. Test case: `> rose84 /tag ABCDEFG` <br>
+       Expected: Contact is not edited. Error details shown in the message box.
+    2. Test case: `> rose84 /tag Finance` <br>
+       Expected: Contact is not edited. Error details shown in the message box. Tags are **case-sensitive**.    
+
```
Comment (ShanylOng) [inline docs/DeveloperGuide.md:497]: LGTM.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -454,32 +455,74 @@ testers are expected to do more *exploratory* testing.
...
 ### Deleting a contact
 
 1. Deleting a person while display is off.
 
    1. Prerequisites: 
       * Clear all contacts using the `clear` command. 
-      * Add a few contacts in according to [this section](#adding-a-person).
+      * Add a few contacts in according to [this section](#adding-a-contact).
```
Comment (ShanylOng) [inline docs/DeveloperGuide.md:504]: Thanks for updating the link. LGTM.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -454,32 +455,74 @@ testers are expected to do more *exploratory* testing.
...
       * Toggle the display using the `$` command, to turn it off.
 
    1. Test case: `- /id johndoe61`<br>
-      Expected: Contact with the ID johndoe61 will be deleted. Details of the deleted contact shown in the status message. Display remains off. 
+      Expected: Contact with the ID johndoe61 will be deleted. Details of the deleted contact shown in the message box. Display remains off. 
 
    1. Test case: `- /id `<br>
-      Expected: No person is deleted. Error details shown in the status message.
+      Expected: No person is deleted. Error details shown in the message box.
```
Comment (ShanylOng) [inline docs/DeveloperGuide.md:512]: Thanks for rewording. LGTM.

**Review by ShanylOng — COMMENTED** [review]
All LGTM. Thanks Zen Khoon.

**Review by ljy0422 — APPROVED** [review]
LGTM!

**Pull request merged**